VL.Data/Model/UserDTO.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.ComponentModel.DataAnnotations;
```

Comment:
Change file name or ReSharper/VS will warn you / the next person, 

`UserDTO.cs` -> `UserDto.cs`